src/persist-client/src/batch.rs

```diff
@@ -490,7 +490,6 @@ impl<T: Timestamp + Codec64> BatchParts<T> {
                 .await
             {
                 Ok(()) => (),
-                Err(err) if err.is_cancelled() => (),
```

Comment:
hmm I wonder why this was here in the first place. I have to assume something with panic'ing during shutdown? maybe it's something that went away when we moved to CpuHeavyRuntime